### ISSUE TITLE
!todo Beaver -Sejant-erect tail stays Grey.

### ISSUE THREAD

**Issue by karlwilcox** [drawshield/Drawshield-Code#649]
!todo Beaver -Sejant-erect tail stays Grey.

**Comment by karlwilcox** [comment]
This was an artistic decision since beaver tails look grey to me! e.g. https://cdn.britannica.com/83/28383-050-33D8DB80/Beaver.jpg?w=400&h=300&c=crop
But willing to change if folks think it is appropriate

**Comment by steve-the-player** [comment]
Wouldn’t that be _a beaver proper_ or _a beaver real_?

**Comment by karlwilcox** [comment]
Agree, the grey tail is inconsistent with the other beavers, so I have changed it.
